package.json

```diff
@@ -36,7 +36,7 @@
   "dependencies": {
     "accessible-autocomplete": "^3.0.0",
     "clipboard": "^2.0.11",
-    "govuk-frontend": "^5.5.0",
+    "govuk-frontend": "github:alphagov/govuk-frontend#0865cb4fb",
```

Comment:
Making a comment so that we don't forget that this needs to be removed before merging